packages/@webex/plugin-meetings/src/roap/turnDiscovery.ts

```diff
@@ -10,6 +10,8 @@ import RoapRequest from './request';
 import Meeting from '../meeting';
 import MeetingUtil from '../meeting/util';
 
+type Enum<T extends Record<string, unknown>> = T[keyof T];
```

Comment:
does this just return whatever the value type of the record is? and unknown by default?

Reply:
it creates a type that has the possible values of the enum - see https://confluence-eng-gpk2.cisco.com/conf/display/WTWC/How+to+use+enums+and+alternatives

Actually I've just noticed that SDK already has this `Enum` type defined in plugin-meetings/src/constants, so I'll just import it from there